vampire_survivors/src/main.py

```
@@ -0,0 +1,343 @@
+import pygame, sys
```

Ruff (0.12.2)

Multiple imports on one line

Split imports

(E401)